terraform/cloudfoundry/cf_rds.tf

```diff
@@ -40,7 +40,7 @@ resource "aws_db_instance" "cf" {
   allocated_storage    = 10
   engine               = "postgres"
   engine_version       = "9.5.4"
-  instance_class       = "db.t2.small"
+  instance_class       = "db.m4.large"
```

Comment:
Even throttled `t2.medium` is sufficient.

But I agree with @keymon, this is bikeshedding as there are many other places where we should look to save costs.